Formula/sse2neon.rb

```diff
@@ -0,0 +1,32 @@
+class Sse2neon < Formula
+  desc "Translator from Intel SSE intrinsics to Arm/Aarch64 NEON implementation"
+  homepage "https://github.com/DLTcollab/sse2neon"
+  url "https://github.com/DLTcollab/sse2neon/archive/v1.5.1.tar.gz"
+  sha256 "4001e2dfb14fcf3831211581ed83bcc83cf6a3a69f638dcbaa899044a351bb2a"
+  license "MIT"
+  head "https://github.com/DLTcollab/sse2neon.git", branch: "master"
+
+  def install
+    include.install "sse2neon.h"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <assert.h>
+      #include <#{Hardware::CPU.intel? ? "emmintrin.h" : "sse2neon.h"}>
```

Comment:
This means it won't test the one installed file on intel though

Reply:
@SMillerDev It can't do, because sse2neon simply cannot be used on Intel.

Reply:
In that case it needs a `depends_on :arm64`

Reply:
Ah okay. I was looking for something like that initially, but it seems it isn't yet used in any homebrew-core formula! Thanks.

Reply:
Nope, this might be the very first ARM only formula

Reply:
@SMillerDev Hah, fair enough. Does the test failure mean anything to you? How do we fix that?